src/datastore/NedbDatastore.ts

```diff
@@ -63,6 +65,14 @@ export class NedbDatastore implements Datastore {
         private readonly teamStore: NedbDb) {
     }
 
+    storeUserActivity(matrixId: string, activity: UserActivity): Promise<void> {
```

Comment:
Decided to give up on these completely since NeDB is already deprecated since a few versions ago.